cmd/libsnap-confine-private/cleanup-funcs-test.c

```diff
@@ -87,7 +87,7 @@ static void test_cleanup_endmntent(void)
 
 	/* It is safe to use with a non-NULL FILE. */
 	GError *err = NULL;
-	g_autofree char *mock_fstab = NULL;
```

Comment:
Technically autofree is just __attribute__((...)) cleanup thing, which has slightly different semantics, but because our failed tests do _not_ abort test execution, this is fine.